### PR TITLE
update crypto-perf

### PR DESCRIPTION
**Pull request by mythi**
Fixes: #1360

### CODE REVIEW

**Review by tkatila — DISMISSED** [review]

File: demo/crypto-perf/run-dpdk-test [hunk 1]
```diff
@@ -10,10 +10,16 @@ fi
...
+if [ -d "${CGROUP_PATH}"/cpuset ]; then
+	CGROUP_PATH="${CGROUP_PATH}"/cpuset
+fi
+
+LCORE=$(cat "${CGROUP_PATH}"/cpuset.cpus)
 
 EAL="--no-telemetry -l ${LCORE} ${PCI_ALLOWLIST}"
 
-dpdk-test-${TESTCMD}-perf ${EAL} -- ${PTEST} \
+dpdk-test-"${TESTCMD}"-perf ${EAL} -- ${PTEST} \
```
[the remark this answers is not in the document]
Comment (hj-johannes-lee) [reply demo/crypto-perf/run-dpdk-test:25]: Aha, ok. Can it be fixed in some other way?
Do we just ignore? Or do we need to improve at some point? Do you want me to find out?

[the remark this answers is not in the document]
Comment (mythi) [reply demo/crypto-perf/run-dpdk-test:25]: I'd keep it as it is now. If you want, I could fix the commit message that not all errors are still fixed

[the remark this answers is not in the document]
Comment (hj-johannes-lee) [reply demo/crypto-perf/run-dpdk-test:25]: OK! :)

**Review by hj-johannes-lee — APPROVED** [review]
LGTM!

**Review by tkatila — APPROVED** [review]

**Pull request merged**